setup.py

```diff
@@ -104,10 +104,11 @@ def find_casacore():
             casacoreversion = getCasacoreVersion()
         except:
             # getVersion was fixed in casacore 2.3.0
-            warnings.warn("Your casacore version is older than 2.3.0! You need to upgrade your casacore.")
```

Comment:
a setup script should never fail, so we should not raise an exception.

Reply:
What should it do? It cannot logically succeed.

Reply:
Ok, I'll print a warning instead.